app.js

```diff
@@ -17,15 +17,18 @@ app.use( bodyParser.json({
 }));
 
 let services = require(CONFIG_SERVICES_JSON_PATH);
+let services_override = require(CONFIG_SERVICES_OVERRIDE_JSON_PATH);
```

Comment:
I think this will crash if you don't specify it. I would suggest to handle this case; we don't always want full override AND fix issues with reverse compatibility